src/Makefile

```diff
@@ -11,10 +11,14 @@ ifeq ($(OS),Windows_NT)
   FSC     = fsc
   MSBUILD = $(shell cygpath -u \"$$(reg.exe query 'HKLM\SOFTWARE\Microsoft\MSBuild\ToolsVersions\14.0' /v MSBuildToolsPath | tail -n 2 | head -n 1 | sed 's/.*REG_SZ\s\+//g')\")/MSBuild.exe
   UNAME   = Windows_NT
+  FSTAR   = ../bin/fstar.exe
+  RUNTIME =
```

Comment:
I thought the same, but it is used in other places in the Makefile (e.g. to run FSYacc FSLex, NuGet).